charts/gardener-dashboard/charts/runtime/templates/dashboard/deployment.yaml

```diff
@@ -0,0 +1,190 @@
+{{- if .Values.global.dashboard.enabled }}
+apiVersion: {{ include "deploymentversion" . }}
+kind: Deployment
+metadata:
+  name: gardener-dashboard
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app.kubernetes.io/name: gardener-dashboard
+    app.kubernetes.io/component: dashboard
+    helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    app.kubernetes.io/instance: "{{ .Release.Name }}"
+    app.kubernetes.io/managed-by: "{{ .Release.Service }}"
+spec:
+  replicas: {{ .Values.global.dashboard.replicaCount }}
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: gardener-dashboard
+      app.kubernetes.io/component: dashboard
+  template:
+    metadata:
+      annotations:
+        checksum/configmap-gardener-dashboard-config: {{ include (print .Template.BasePath "/dashboard/configmap.yaml") . | sha256sum }}
+        checksum/secret-gardener-dashboard-sessionSecret: {{ include (print .Template.BasePath "/dashboard/secret-sessionSecret.yaml") . | sha256sum }}
+        {{- if .Values.global.dashboard.oidc }}
+        checksum/secret-gardener-dashboard-oidc: {{ include (print .Template.BasePath "/dashboard/secret-oidc.yaml") . | sha256sum }}
+        {{- end }}
+        {{- if .Values.global.dashboard.github }}
+        checksum/secret-gardener-dashboard-github: {{ include (print .Template.BasePath "/dashboard/secret-github.yaml") . | sha256sum }}
+        {{- end }}
+        {{- if .Values.global.dashboard.kubeconfig }}
+        checksum/secret-gardener-dashboard-kubeconfig: {{ include (print .Template.BasePath "/dashboard/secret-kubeconfig.yaml") . | sha256sum }}
+        {{- end }}
+      labels:
+        app.kubernetes.io/name: gardener-dashboard
+        app.kubernetes.io/component: dashboard
+        helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+        app.kubernetes.io/instance: "{{ .Release.Name }}"
+        app.kubernetes.io/managed-by: "{{ .Release.Service }}"
+        {{- if .Values.global.dashboard.podLabels }}
```

Comment:
do you want to add it to the `values.yaml` as documentation? like
`# podLabels: # YAML formatted labels used for pod template`
Same as for the bootstrapper deployment

Reply:
✔️